Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+# v0.12.4
+- [fixed] DocumentReference.setData(_:mergeFields:) can now delete fields using
```

Comment:
Feel free to leave as is, but this applies to WriteBatch and Transaction as well.

Reply:
Fixed.